core-bundle/src/Resources/contao/classes/FrontendUser.php

```diff
@@ -248,6 +250,27 @@ protected function setUserFromDb()
 		}
 	}
 
+	/**
+	 * @param array|int|string $ids
+	 * @return bool
+	 *
+  	 * @deprecated Deprecated since Contao 4.12, to be removed in Contao 5.0.
+	 *             Use Symfony security instead.
+	 */
+	public function isMemberOf($ids)
```

Comment:
Since you are no longer using the `isMemberOf()` method in the `MemberGroupVoter`, I guess we do not have to add this method at all, do we?

Reply:
huh? That's for anyone doing the same permissions checks we previously did.

Reply:
Why not add a `USER_IN_GROUPS` voter and deprecate the method on the `User` object?